MaterialSkin/HTML/material/html/js/browse-page.js

```diff
@@ -1595,17 +1597,53 @@ var lmsBrowse = Vue.component("lms-browse", {
         },
         setScrollElement() {
             this.scrollElement = document.getElementById(this.useGrid ? "browse-grid" : "browse-list");
+            var isScrolling;
             this.scrollElement.addEventListener('scroll', () => {
-                if (this.fetchingItems || this.listSize<=this.items.length) {
-                    return;
-                }
+                // display a letter as overlay whilst scrolling - displays first letter of item at top of screen
+
+                window.clearTimeout(isScrolling);                            // Clear a timeout if still scrolling
+                isScrolling =  setTimeout( () => {                           // Show the overlay for 0.5 sec after scrolling...
+                               this.showLetter = false;
+                               },500);
+
                 const scrollY = this.scrollElement.scrollTop;
                 const visible = this.scrollElement.clientHeight;
                 const pageHeight = this.scrollElement.scrollHeight;
                 const pad = (visible*2.5);
 
                 const bottomOfPage = (visible + scrollY) >= (pageHeight-(pageHeight>pad ? pad : 300));
 
+                const numPages =  Math.floor(scrollY/visible)+1;            // Calculate number of pages scrolled
+                var stopWordsRE = /(?:^|\s+)(?:the|The|El)(?=\s+|$)/gi;     // Pathetic attempt at stop word removal (needs to be a parameter)
+	 
+ 
+                if (this.useGrid) {                                         // Grid Display, 
+                    var elem = document.getElementById("item0");
+                    var rows = Math.round( scrollY / elem.offsetHeight); 
+                    var colsPerPage  =   Math.floor(this.scrollElement.clientWidth / elem.offsetWidth) ; 
+                    var itemNum = "item"+ Math.floor(rows*colsPerPage);
+                    var classname = "v-card v-card--flat v-card--tile theme--" + ( this.$store.state.darkUi ? "dark" : "light"); 
+                    elem = document.getElementById(itemNum).getElementsByClassName(classname)[0];
+                    var  overlayHTML = elem.title.replace(stopWordsRE,"").trim().charAt(0).toUpperCase();
+		}else {                                                     // List display 
+                    var classname = "v-list__tile v-list__tile--link v-list__tile--avatar theme--" + (this.$store.state.darkUi ? "dark" : "light");
```

Comment:
Ummm... this looks a bit odd here. Not sure I like this. why by class name?

Reply:
So for the GridView individual items are have ids of the form "item01", "item02" etc
But from what I read, the card text showing title doesn't have an id -  so needs to be retrieved via a classname? Or am I misreading?

Reply:
No problems - thanks for your insights. Can I check my understanding on Grid view?

Each grid item has a key of item0 etc
An example title div inside the grid item is specified as 
`<div title="1 (2000)The Beatles" class="v-card v-card--flat v-card--tile theme--dark">`

I couldn't see how to access it without using the class name.  I guess solvable by generating an ID for the title div? Just want to check my understanding. thanks for your help.